### PR TITLE
New package: kurogame.PunishingGrayRaven version 1.0.0.4

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/110762&drop=dogfoodAlpha

### CODE REVIEW

**Comment by stephengillie** [comment]
This package appears to be a tray-based application that pops up into an always on top window, despite having no executables detected. 
image

File: manifests/k/kurogame/PunishingGrayRaven/1.0.0.4/kurogame.PunishingGrayRaven.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: kurogame.PunishingGrayRaven
+PackageVersion: 1.0.0.4
+PackageLocale: en-US
+Publisher: Guangzhou Kuro Technology Co., Ltd.
+PackageName: 'Punishing: Gray Raven'
+License: Proprietary
+Copyright: ©KURO TECHNOLOGY (HONG KONG) CO.,LIMITED
+ShortDescription: 'Punishing: Gray Raven is a 3D post-apocalyptic and sci-fi action RPG'
```
Comment (stephengillie) [inline manifests/k/kurogame/PunishingGrayRaven/1.0.0.4/kurogame.PunishingGrayRaven.locale.en-US.yaml:11]: This package appears to be a math equation input program, not a 3d RPG.

**Comment by solomoncyj** [comment]
this was not supposed to be the program

**Comment by solomoncyj** [comment]
https://drive.google.com/file/d/1sj2yyDZYNzmtuQBdBU_5-GgwseDMr55s/view?usp=sharing this was supposee to be the app

**Comment by stephengillie** [comment]
The new Math panel above confused me before. This package appears to do nothing when given the `/S` switch, which might seem similar to the Nullsoft silent install. But with any other switch combination, it launches an interactive install:
image

**Pull request closed without merging**